README.md

```diff
@@ -1 +1,1178 @@
-# ballerina-honeycomb
\ No newline at end of file
+  [![Build Status](https://travis-ci.org/ballerina-guides/ballerina-honeycomb)](https://travis-ci.org/ballerina-guides/ballerina-honeycomb)
+  
+## Integration with Honeycomb
+
+ Honeycomb is a tool used for investigating on how well your system product is working in various conditions (for example - high traffic). Through honeycomb we are able to collect data of your own software which can be broken down into various entities to observe its performance specifically. [Read more about Honeycomb.](#about-honeycomb)
```

Comment:
Remove the extra space before the paragraph. 

Change to: "Honeycomb is a tool used to investigate how well your system works in various conditions (for example - high traffic). Through Honeycomb, you can collect data pertaining to your software that can be broken down into various entities. You can observe the performance of each of these entities specifically."